src/sentry/snuba/metrics/query_builder.py

```diff
@@ -644,10 +652,21 @@ def translate_results(self):
                 else:
                     operation = None
                     metric_name = key
+
                 if (operation, metric_name) not in self._query_definition_fields_set:
                     if totals is not None:
                         del totals[key]
                     if series is not None:
                         del series[key]
+                else:
+                    reversed_mri_key = (
+                        f"{get_reverse_mri(metric_name)}"
+                        if operation is None
+                        else f"{operation}({get_reverse_mri(metric_name)})"
```

Comment:
Same concern as before: To make the code easier to understand, I would rename `get_reverse_mri()` to `mri_to_public_name()` or something like that.

Reply:
Yep